packages/components/src/components/Address/Address.test.tsx

```diff
@@ -5,7 +5,7 @@ import { Address } from './Address';
 
 describe('Address', () => {
   it('renders without throwing', () => {
-    const { container } = render(<Address value="taylorswift.eth" />);
+    const { container } = render(<Address value='taylorswift.eth' shortened={false} />);
```

Comment:
You also won't need to set shortened={false} once you make that prop optional

```suggestion
    const { container } = render(<Address value='taylorswift.eth' />);
```